### DISCUSSION TITLE
Checking if running WinterCMS or OctoberCMS

### DISCUSSION THREAD

**Discussion by ericp-mrel** [wintercms/winter#181] [Q&A] [answered]
Is there currently a good way to be able to check from a plugin if it is running in Winter CMS vs October CMS in order to be able to adjust logic where needed?

**Comment by LukeTowers** [comment, marked as answer]
Check if a Winter class exists would be my recommendation.

**Reply by ericp-mrel** [reply]
Yeah. that's what I was thinking. Any recommendations on what class you think would be good to check for that would be fairly stable and not removed for the foreseeable future?

Right now, I'm checking for the `\Winter\Storm\Foundation\Application`  class since that seems like something that shouldn't change, if at all.

**Reply by bennothommo** [reply]
That should be fine to use. That file will never change.
